packages/syft/src/syft/orchestra.py

```diff
@@ -183,7 +183,9 @@ def deploy_to_python(
     log_level: str | int | None = None,
     debug: bool = False,
     migrate: bool = False,
+    store_client_config: dict | None = None,
```

Comment:
is this convenient to pass, do we have an example of how we would pass it? Also, how do I know the default value?

Reply:
it's old code and should be removed

Reply:
Removed [3b3023d](https://github.com/OpenMined/PySyft/pull/9176/commits/3b3023d3aa3f3a9914b276a0663f28c4feebe0a0)